spk/filebrowser/Makefile

```
@@ -0,0 +1,27 @@
+SPK_NAME = filebrowser
+SPK_VERS = 2.17.1
+SPK_REV = 1
+SPK_ICON = src/filebrowser.png
+
+DEPENDS = cross/$(SPK_NAME)
+UNSUPPORTED_ARCHS = $(PPC_ARCHS)
+
+MAINTAINER = publicarray
+DESCRIPTION = Web File Browser
+DISPLAY_NAME = File Browser
+HELPURL = https://github.com/filebrowser/filebrowser/issues
+LICENSE = Apache-2.0
+CHANGELOG = "First Version"
+
+SPK_GROUP=synocommunity
+GROUP=synocommunity
```

just for the sake of consistency..
```suggestion
SPK_GROUP = synocommunity
GROUP = synocommunity
```

---

Thanks :+1: I want to merge #4789 then having both is no longer necessary.

#4789 requires testing since the group in DSM7 behaves differently then in DSM6. From memory if the group in DSM6 is created and the upgraded to DSM7 it is created again, and the new group is renamed to something like `groupname__PKG__` because it already exists. But I might be wrong and unsure how to test this.